feature-libs/cart/base/components/cart-shared/cart-item-list-row/cart-item-list-row.component.html

```diff
@@ -10,6 +10,7 @@
     <div class="cx-table-item-container">
       <!-- Item Image -->
       <a
+        *ngIf="!options.disableItemLink"
```

Comment:
for info: 
not considered as breaking change as added property is optional and DOM structure stayed unchanged for existing customers.